### DISCUSSION TITLE
Showing Scroll Bar At The Top Of Table

### DISCUSSION THREAD

**Discussion by BenOussama180** [Power-Components/livewire-powergrid#656] [Q&A]
Is there a way to show the scroll bar for large tables in the top and in the bottom of the table instead of only iin the bottom  ? sometiimes the table iis way too big ( showiing 50 results ) we need to scrool down to use scrool bar

**Comment by luanfreitasdev** [comment]
Hello.

Do you need something like this?

https://user-images.githubusercontent.com/33601626/194712394-307b0b62-03c8-4625-8243-8196bfd28c68.mp4

https://github.com/Power-Components/powergrid-demo-misc/blob/main/app/Http/Livewire/HeaderFixedTable.php
https://github.com/Power-Components/powergrid-demo-misc/blob/main/app/Http/Livewire/PowerGridThemes/TailwindHeaderFixed.php

**Reply by BenOussama180** [reply]
no im talking about horizontal scroll, i have tables with do many fields so for the user to scroll right or left he have to go down on the table to use the scroll bar , is there a way to show scroll bar at the top ? horizontally

**Reply by luanfreitasdev** [reply]
It currently doesn't. But it's an important point if it's possible
